src/main/io/vtx_control.h

```diff
@@ -28,7 +28,9 @@
 #include "pg/pg.h"
 #include "fc/rc_modes.h"
 
+#ifndef MAX_CHANNEL_ACTIVATION_CONDITION_COUNT
```

Comment:
When testing we get an build error without log, seems there is a maximum length allowed (around length 27) per define, needs to be renamed in a few other places too.

Reply:
So `MAXCHANACTIVATIONCONDITIONC` will ensure value = 99 max :stuck_out_tongue: